dmutils/request_id.py

```diff
@@ -9,6 +9,8 @@ class RequestIdRequestMixin(object):
         A mixin intended for use against a flask Request class, implementing extraction (and partly generation) of
         headers approximately according to the "zipkin" scheme https://github.com/openzipkin/b3-propagation
     """
+    # a single class-wide random instance should be good enough for now
```

Comment:
I don't really follow the "for now" aspect here. If there's no conceivable reason to change this, then don't say "for now". If there is a reason this maybe should change one day, then call out the specific?

Reply:
God knows, we might find using the `SystemRandom` to be overkill for this purpose and it sucks up our instance's entropy. Don't know. But it's a comment.